pkgs/by-name/ni/nix-fast-build/package.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  nix-eval-jobs,
+  nix-output-monitor,
+  nix-update-script,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "nix-fast-build";
+  version = "1.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Mic92";
+    repo = "nix-fast-build";
+    rev = "refs/tags/${version}";
+    hash = "sha256-8zW6eWvE9T03cMpo/hY8RRZIsSCfs1zmsJOkEZzuYwM=";
+  };
+
+  build-system = [ python3Packages.setuptools ];
+
+  makeWrapperArgs = [
+    "--prefix PATH : ${
+      lib.makeBinPath [
+        nix-eval-jobs
+        nix-eval-jobs.nix
+        nix-output-monitor
+      ]
+    }"
+  ];
+
+  nativeCheckInputs = [ python3Packages.pytestCheckHook ];
+
+  disabledTests = [
+    # fail due to using nix
+    "test_build"
+    "test_eval_error"
+    "test_remote"
+  ];
```

In my opinion python tests are mainly useful to ensure that python deps work properly together. However in this case the project doesn't use any third-party modules and I test all releases in the upstream CI on both linux/macOS with nix.
 
```suggestion
  # Tries to run nix to build stuff, which we cannot do inside the sandbox.
  doCheck = false;
```